### PR TITLE
QE: Add Tests for Left Menu Filter

### PR DESCRIPTION
**Pull request by elariekerboull**
### Implement  left menu filter tests
Added automatic test scenarios:

- One correct search result with corresponding word
- One correct search result with partial corresponding word
- One correct search result with different cased word
- Several correct search results with matching word
- No search result with non-present word

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage

- Cucumber tests were added

- [x] **DONE**

## Links

Fixes # [7868](https://github.com/SUSE/spacewalk/issues/7868)
- 4.2 https://github.com/SUSE/spacewalk/pull/17451
- 4.1 https://github.com/SUSE/spacewalk/pull/17486

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"   
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [x] Re-run test "ruby_rubocop"   
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/features/secondary/srv_menu_filter.feature [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  Scenario: Search a partial word and expect the right result
+    When I enter "Formula" as the left menu search field
+    Then I should see a "Formula Catalog" link in the left menu
+
+  Scenario: Search a word with a different case and expect the right result
+    When I enter "saLT" as the left menu search field
+    Then I should see a "Salt" link in the left menu  
+
+@susemanager
```
Comment (srbarrios) [inline testsuite/features/secondary/srv_menu_filter.feature:25]: I only now notice that, I would say this is bad indented, I think need to be at same indentation that the scenario title.
But I'm not totally sure. In any case, fine to merge this port for now. If it did not failed in 4.2 CI

Comment (srbarrios) [reply testsuite/features/secondary/srv_menu_filter.feature:25]: Let's fix it here @elariekerboull , and then we port this to other branches.
Sorry to catch this a bit late.

Comment (nodeg) [reply testsuite/features/secondary/srv_menu_filter.feature:25]: I had another look at our code and it seems we have a mixed indentation regarding the tags.

**Review by srbarrios — APPROVED** [review]
Thanks!

**Pull request merged**